Source/Events/ReplicateItemListUpdate.lua

```diff
@@ -17,14 +18,20 @@ function Auctionator.Events.ReplicateItemListUpdate()
     local prices = {}
 
     for index = 0, C_AuctionHouse.GetNumReplicateItems() - 1 do
-      local _, _, count, _, _, _, _, _, _, buyoutPrice, _, _, _, _, _, _,
+      local name, _, count, _, _, _, _, _, _, buyoutPrice, _, _, _, _, _, _,
```

Comment:
I think we can use `select` to make this a little more concise? (Could be wrong, not sure if we would have to `select` once for each variable we want, and if that means we are calling the `GetReplicateItemInfo` multiple times?)